frontend/apps/reader/modules/readerfont.lua

```diff
@@ -141,7 +141,7 @@ end
 function ReaderFont:onGesture() end
 
 function ReaderFont:registerKeyEvents()
-    if Device:hasKeyboard() then
+    if Device:hasKeyboard() and not Device:hasFiveWay() then
         -- add shortcut for keyboard
         self.key_events = {
             ShowFontMenu = { { "F" } },
```

Comment:
@poire-z Incidentally we overlooked this in the font menu changes (press `f`). :-) Changing the fonts works but the submenus are a bit hit and miss.

<img src=https://github.com/koreader/koreader/assets/202757/89d73f15-3114-4366-bc8c-19e8bd041ef5 width=50%>

Reply:
Okay, I think I understood the intent. The `F` was an accidental victim. Resolved like this:

```diff
diff --git a/frontend/apps/reader/modules/readerfont.lua b/frontend/apps/reader/modules/readerfont.lua
index d79296a38..054ca8d3c 100644
--- a/frontend/apps/reader/modules/readerfont.lua
+++ b/frontend/apps/reader/modules/readerfont.lua
@@ -141,21 +141,21 @@ end
 function ReaderFont:onGesture() end
 
 function ReaderFont:registerKeyEvents()
-    if Device:hasKeyboard() and not (Device:hasDPad() and Device:useDPadAsActionKeys()) then
-        -- add shortcut for keyboard
-        self.key_events = {
-            ShowFontMenu = { { "F" } },
-            IncreaseSize = {
+    if Device:hasKeyboard() then
+        self.key_events.ShowFontMenu = { { "F" } }
+        if not Device:hasScreenKB() then
+            -- add shortcut for keyboard
+            self.key_events.IncreaseSize = {
                 { "Shift", Input.group.PgFwd },
                 event = "ChangeSize",
                 args = 0.5
-            },
-            DecreaseSize = {
+            }
+            self.key_events.DecreaseSize = {
                 { "Shift", Input.group.PgBack },
                 event = "ChangeSize",
                 args = -0.5
-            },
-        }
+            }
+        end
     end
 end
```

Reply:
This menu is more something bothering I get when I accidently hit <kbd>f</kbd>...
I would just kill it - why a dedicated menu for fonts, and not for everything else ?
I think we don't have any other instance of this kind of use of Menu.

But if you want to keep it, what does it miss from the "font menu changes" ? The other items Font settings and Font family fonts ? As a quick "change font" menu, I don't think it needs them.

Reply:
I don't specifically want to keep it, no. So I'll just remove it?

Reply:
Fine with me :)
People who were used to it can probably add it with a userpatch, we'll help if someone complain about missing it.